src/io_soak/io_soak_test.go

```
@@ -63,7 +63,11 @@ func monitor() error {
 				logf.Log.Info("IOSoakTest monitor Mayastor volumes check: Ignoring", "error", err)
 			} else {
 				logf.Log.Info("IOSoakTest monitor Mayastor volumes check", "error", err)
-				break
+				// FIXME: do not consider volume health a test failure. When running this test on shared resources
```

I'd be tempted to set err to nil here to make it clear that the error is being ignored.
Also if the code ever gets refactored so that this block is the last to set err in the loop, err will no longer be suppressed.

---

On the other hand that would probably trigger the "ineffectual assignment" lint error.

---

Unfortunately doing so falls foul of the linters
```
INFO [runner] linters took 1.060330943s with stages: goanalysis_metalinter: 699.439293ms, unused: 359.782953ms 
io_soak/io_soak_test.go:70:5: ineffectual assignment to `err` (ineffassign)
                                err = nil
                                ^
```